src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  border: none;
+  outline: none;
+}
+
+/* Both search forms styles */
+
+.container {
+  margin-top: 20px;
+}
+
+.image {
+  position: absolute;
+}
+
+.input {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+/* Large search form styles  */
+
+.large__container {
+  height: 70px;
+}
+
+.large__image {
+  top: 45px;
+  left: 34px;
+}
+
+.large__input {
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 800;
+}
+
+.large__input:focus {
+  font-weight: 800;
+}
+
+.large__input:focus::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
```

Comment:
No need to create modifiers for both small and big inputs, just write right styles to generic class, and override sizes for small elements

Reply:
Fix for all classes